doc/r1cs_bin_format.md

```diff
@@ -146,7 +146,7 @@ Section Type: 0x01
      ┃ 4  │   20 00 00 00   ┃               Field Size in bytes (fs)
      ┗━━━━┻━━━━━━━━━━━━━━━━━┛
      ┏━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
-     ┃ fs │   010000f0 93f5e143 9170b979 48e83328 5d588181 b64550b8 29a031e1 724e6430 ┃  Prime size
+     ┃ fs │   010000f0 93f5e143 9170b979 48e83328 5d588181 b64550b8 29a031e1 724e6430 ┃  Prime
      ┗━━━━┻━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
      ┏━━━━┳━━━━━━━━━━━━━━━━━┓
      ┃ 32 │   01 00 00 00   ┃               nWires
```

Comment:
Another issue is that the field size field uses `bytes` as unit, while `nWires` and the rest use bits.